erpnext_shipping/erpnext_shipping/utils.py

```diff
@@ -71,7 +71,7 @@ def validate_phone(doc, method=None):
 	if not phone_number:
 		frappe.throw(_("Pickup contact phone is required."))
 
-	if not re.match(r"^\+(?!0)\d+$", phone_number):
+	if not re.match(r"^\+(?!\s*0)[\d\s]+$", phone_number):
```

Comment:
```suggestion
	if not re.match(r"^\+(?![\s0])[\d\s]+\d$", phone_number):
```

- Disallow whitespace after the "+"
- Number must end with a digit, not a whitespace